cranelift/filetests/filetests/pcc/succeed/fuzz-float-loads.clif

```diff
@@ -1,86 +0,0 @@
-test compile
-set enable_pcc=true
-set opt_level=speed
-target x86_64
-
-function u0:0(i64 vmctx, i64) fast {
-    gv0 = vmctx
-    gv1 = load.i64 notrap aligned readonly gv0+8
-    gv2 = load.i64 notrap aligned gv1
-    gv3 ! mem(mt0, 0x0, 0x0) = vmctx
-    gv4 ! mem(mt1, 0x0, 0x0) = load.i64 notrap aligned readonly checked gv3+80
-    mt0 = struct 88 { 80: i64 readonly ! mem(mt1, 0x0, 0x0) }
-    mt1 = memory 0x180000000
-    sig0 = (i64 vmctx) system_v
-    sig1 = (i64 vmctx, i32 uext) system_v
-    sig2 = (i64 vmctx, i32 uext, i32 uext, i32 uext) -> i32 uext system_v
-    const0 = 0x02030077ff00ff04030077ff6cff5503
-    stack_limit = gv2
-
-block0(v0: i64, v1: i64):
-    v2 = iconst.i32 0
-    v25 -> v2
-    v45 -> v2
-    v24 -> v25
-    v3 = null.r64
```

Comment:
It looks like this whole test was deleted because it uses a `null.r64` instruction, but the instruction isn't critical to the test (result of random fuzz generation); could we replace it with an `iconst.i64 0` or somesuch and keep the test?

Reply:
Ah, I didn't look closely and assumed it relied on the `r64`; added it back and updated it!